src/grasp_scorer.cpp

```diff
@@ -110,6 +110,89 @@ Eigen::Vector3d GraspScorer::scoreGraspTranslation(const Eigen::Affine3d& grasp_
+  Eigen::Affine3d gripper_corner_tr = Eigen::Affine3d::Identity();
+  Eigen::Affine3d gripper_corner_tl = Eigen::Affine3d::Identity();
+  Eigen::Affine3d gripper_corner_br = Eigen::Affine3d::Identity();
+  Eigen::Affine3d gripper_corner_bl = Eigen::Affine3d::Identity();
+
+  Eigen::Affine3d box_corner_tr = Eigen::Affine3d::Identity();
+  Eigen::Affine3d box_corner_tl = Eigen::Affine3d::Identity();
+  Eigen::Affine3d box_corner_br = Eigen::Affine3d::Identity();
+  Eigen::Affine3d box_corner_bl = Eigen::Affine3d::Identity();
+
+  gripper_corner_tr.translation() =
+      Eigen::Vector3d(grasp_data->active_suction_range_x_ / 2.0, grasp_data->active_suction_range_y_ / 2.0, 0.0);
+  gripper_corner_tl.translation() =
+      Eigen::Vector3d(grasp_data->active_suction_range_x_ / 2.0, -grasp_data->active_suction_range_y_ / 2.0, 0.0);
+  gripper_corner_br.translation() =
+      Eigen::Vector3d(-grasp_data->active_suction_range_x_ / 2.0, grasp_data->active_suction_range_y_ / 2.0, 0.0);
+  gripper_corner_bl.translation() =
+      Eigen::Vector3d(-grasp_data->active_suction_range_x_ / 2.0, -grasp_data->active_suction_range_y_ / 2.0, 0.0);
+
+  box_corner_tr.translation() = Eigen::Vector3d(object_size[0] / 2.0, object_size[1] / 2.0, 0.0);
+  box_corner_tl.translation() = Eigen::Vector3d(object_size[0] / 2.0, -object_size[1] / 2.0, 0.0);
+  box_corner_br.translation() = Eigen::Vector3d(-object_size[0] / 2.0, object_size[1] / 2.0, 0.0);
+  box_corner_bl.translation() = Eigen::Vector3d(-object_size[0] / 2.0, -object_size[1] / 2.0, 0.0);
+
+  gripper_corner_tr = object_pose.inverse() * grasp_pose * gripper_corner_tr;
+  gripper_corner_tl = object_pose.inverse() * grasp_pose * gripper_corner_tl;
+  gripper_corner_br = object_pose.inverse() * grasp_pose * gripper_corner_br;
+  gripper_corner_bl = object_pose.inverse() * grasp_pose * gripper_corner_bl;
+
+  double box_max_tx = std::max(box_corner_tr.translation().x(), box_corner_tl.translation().x());
+  double box_max_bx = std::max(box_corner_br.translation().x(), box_corner_bl.translation().x());
+  double box_max_x = std::max(box_max_tx, box_max_bx);
+
+  double box_min_tx = std::min(box_corner_tr.translation().x(), box_corner_tl.translation().x());
+  double box_min_bx = std::min(box_corner_br.translation().x(), box_corner_bl.translation().x());
+  double box_min_x = std::min(box_min_tx, box_min_bx);
+
+  double box_max_ty = std::max(box_corner_tr.translation().y(), box_corner_tl.translation().y());
+  double box_max_by = std::max(box_corner_br.translation().y(), box_corner_bl.translation().y());
+  double box_max_y = std::max(box_max_ty, box_max_by);
+
+  double box_min_ty = std::min(box_corner_tr.translation().y(), box_corner_tl.translation().y());
+  double box_min_by = std::min(box_corner_br.translation().y(), box_corner_bl.translation().y());
+  double box_min_y = std::min(box_min_ty, box_min_by);
+
+  double gripper_max_tx = std::max(gripper_corner_tr.translation().x(), gripper_corner_tl.translation().x());
```

Comment:
`std::min` / `std::max` accept initializer lists.
```suggestion
  double gripper_max_x = std::max({gripper_corner_tr.x(),
                                   gripper_corner_tl.x(), 
                                   gripper_corner_br.x(),
                                   gripper_corner_bl.x()});
```

Reply:
implemented